sync/src/orphan_block_pool.rs

```diff
@@ -12,13 +12,15 @@ const SHRINK_THRESHOLD: usize = 100;
 pub struct OrphanBlockPool {
     blocks: DashMap<ParentHash, HashMap<packed::Byte32, core::BlockView>>,
```

Comment:
(Minor suggestion) Add a comment:

```suggestion
    // Group by blocks in the pool by the parent hash.
    blocks: DashMap<ParentHash, HashMap<packed::Byte32, core::BlockView>>,
```